pybossa/view/projects.py

```diff
@@ -1364,7 +1365,7 @@ def respond_export(download_type, args):
                 task_run = TaskRun(project_id=project.get('id'))
                 ensure_authorized_to('read', task_run)
 
-            export_queue.enqueue(export_tasks,
+            export_queue.enqueue(partial(export_tasks, disclose_gold=can_know_task_is_gold),
```

Comment:
Seems can_know_task_is_gold to be set to true for admins and subadmin proj co-owners; [here](https://github.com/bloomberg/pybossa/pull/306/files#diff-8fa3bf883c1a925427ca6a3b4587f452R1273)

Reply:
That's right. I don't know what you want me to do. Are you just asking for clarification?

Reply:
Let's confirm w/ stakeholder and update code accordingly.